tpl/tplimpl/embedded/templates/opengraph.html

```diff
@@ -1,14 +1,14 @@
 <meta property="og:url" content="{{ .Permalink }}">
 
-{{- with or site.Title site.Params.title | plainify }}
+{{- with or site.Title site.Params.title | .RenderString | plainify | htmlUnescape }}
   <meta property="og:site_name" content="{{ . }}">
```

Comment:
Nit: could we unindent all of the "  <meta...>" lines (lines 4, 8, 12 here, and the other file) to match the first line of this file that is not indented.

Otherwise, as illustrated in #12418, we get HTML generated like this:


```html
<meta name="description" content="Test line with a single quote: can&amp;rsquo;t.">
<meta property="og:url" content="http://localhost:1313/docs/page/">
  <meta property="og:site_name" content="My New Hugo Site">
  <meta property="og:title" content="Some page">
  <meta property="og:description" content="Test line with a single quote: can&amp;rsquo;t.">
  ...
<meta name="twitter:description" content="Test line with a single quote: can&rsquo;t.">
```

Reply:
Given that you (almost always?) minify in production this definitely seems like a nit. I'd prefer indented code to improve template readability per #10900.

Reply:
FYI, my use case is this: I diff non-minified generated site files of the websites I work on so that I can catch bugs like #12418 (it's impossible to diff minified site files).

In the case of this PR, we can have our cake and eat it too: satisfy #10900 and use appropriate whitespace trimming directives. My 2 cents. Thanks for giving this some thought.

Reply:
Per bep's comment above...

> People can create their own templates if they need special things.

My view on many of the embedded templates is that they are minimal examples, and every time we touch one somebody gets upset. I'd rather eliminate many of them and push responsibility back to site and theme authors. But I guess that train left the station too long ago.

If I were you I would override the opengraph, schema, and twitter_card templates. Based on bep's review, most of the changes in this PR will be eliminated, focusing instead on this one problem (double-escaping HTML entities in the description field of the opengraph template).

Reply:
I totally get it. No worries, I'll roll with what you provide, thanks!